src/extra/AutoUi/Collection/Filters.tsx

```diff
@@ -24,6 +26,47 @@ export const Filters = <T extends AutoUIBaseResource<T>>({
 	renderMode,
 }: FiltersProps<T>) => {
 	const history = useHistory();
+	const { analyticsClient } = useAnalytics();
+
+	React.useEffect(() => {
+		if (!analyticsClient || !filters?.length) {
```

Comment:
Probably it'd be useful to add a similar heuristic like @laziob added on the number of characters in the search box. So we don't send an event for each single character typed in the search bar.